### DISCUSSION TITLE
edge artefact pretrained model

### DISCUSSION THREAD

**Discussion by mhoibo** [tensorflow/compression#191] [Q&A]
Hi, 

I have tried the pre-trained models to compress and decompress images, as well as tried to train one of the models from the model dictionary. 

For all of the pre-trained models, except the hific, and the model I trained, I got an edge "artefact" at the right and bottom edge of the decompressed images. Those edges/sides were dark grey, like a thin, dark grey, fussy frame/line on those sides. I tried images of different sizes and it happened to all of them. 

Does anyone know why this happens (some padding for ex?)? Thank you in advance!
